From bcc2d15362c9622187ab84c1ef89637e34bcd77d Mon Sep 17 00:00:00 2001
From: Jaehoon Chung <jh80.chung@samsung.com>
Date: Mon, 10 May 2021 16:04:02 +0900
Subject: [PATCH 1/1] doc: usage: modify mmc man-page

Modify mmc man-page.
- Change a description about mmc erase command.
- Add whitespace to distinguish.

Signed-off-by: Jaehoon Chung <jh80.chung@samsung.com>
Reviewed-by: Bin Meng <bmeng.cn@gmail.com>
---
 doc/usage/mmc.rst | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/usage/mmc.rst b/doc/usage/mmc.rst
index 57284ed674..458c7642c0 100644
--- a/doc/usage/mmc.rst
+++ b/doc/usage/mmc.rst
@@ -40,7 +40,7 @@ The 'mmc write' command writes raw data to MMC device from memory address with b
     cnt
         block count
 
-The 'mmc erase' command erases MMC device from block offset until count.
+The 'mmc erase' command erases *cnt* blocks on the MMC device starting at block *blk#*.
 
     blk#
         start block offset
@@ -110,6 +110,7 @@ The 'mmc partconf' command shows or changes PARTITION_CONFIG field.
         partitions to access
 
 The 'mmc bootpart-resize' command changes sizes of boot and RPMB partitions.
+
     dev
         device number
     boot part size MB
-- 
2.30.2

